### DISCUSSION TITLE
Testing dependencies step on CI

### DISCUSSION THREAD

**Discussion by ilya-lavrenov** [Homebrew/discussions#4707] [Writing Formulae/Casks] [answered]
### Output of `brew config`

```shell
Not applicable
```

### Output of `brew doctor`

```shell
Not applicable
```

### Description of issue

The PR https://github.com/Homebrew/homebrew-core/pull/134935

Here I update OpenVINO formula with new version. All OpenVINO versions update `SOVERSION` of libraries.
But brew tries to test OpenCV (dependency) compiled with previous version of OpenVINO (currently available in brew) with newer version with changes ABI. 

What is the solution here? How to enforce dependency rebuild? 
Not all libraries are BW binary compatible and change their ABI.

**Comment by ZhongRuoyu** [comment, marked as answer]
`opencv` needs a revision bump. Do this with `brew bump-revision` ([manpage](https://docs.brew.sh/Manpage#bump-revision-options-formula-)); see https://github.com/Homebrew/homebrew-core/pull/135993 for an example.
